tests/foreman/ui/test_reporttemplates.py

```diff
@@ -230,54 +231,41 @@ def test_positive_end_to_end(session, module_org, module_location):
 
 @pytest.mark.upgrade
 @pytest.mark.tier2
-def test_positive_generate_registered_hosts_report(target_sat, module_org, module_location):
+def test_positive_generate_registered_hosts_report(
+    session, target_sat, setup_content, rhel7_contenthost
+):
```

Comment:
Let's get rid of `rhelX_contenthost` in favor of `rhel_contenthost` and parametrize what version we want to test with. Here I would say we could test with all supported versions (as suggested bellow), or pick one at least.
```suggestion
@pytest.mark.rhel_ver_match(r'^(?!6$)\d+$')
def test_positive_generate_registered_hosts_report(
    session, target_sat, module_setup_content, rhel_contenthost
):
```
I can see the `setup_content` fixture is module-scoped - good, we need it module-scoped so we don't create new and new content for every parametrized run, just consider renaming it to what it is - `module_setup_content`.